is_core/static/is_core/css/app.css

```diff
@@ -8959,6 +8959,12 @@ form .required-hint {
     clear: both
 }
 
+form .required-hint {
```

Comment:
Proč je tu ten styl dvakrát (a nastavuje stejné hodnoty atributy stylů)?

Reply:
jednou je to sass podruhé css. Css se generuje ze sass

Reply:
Asi si nerozumíme - stul .required-hint je dvakrát nad sebou v tom app.css souboru, ne v různých souborech. Nebo tím chceš říct, že se ten styl .required-hint vygeneroval dvakrát se sass nebo byl v css a byl doplněn generováním ze sass?

Reply:
aha vidíš už ti rozumím a ano to je fakt divný

Reply:
možná jsem si s tím css pak hrál ručně :D

Reply:
Se stane - tak hlavně že jsme se domluvili :)